exec/java-exec/src/main/resources/rest/query/result.ftl

```diff
@@ -11,14 +11,13 @@
 
 <#include "*/generic.ftl">
 <#macro page_head>
-  <link rel="stylesheet" type="text/css" href="//code.jquery.com/ui/1.10.3/themes/smoothness/jquery-ui.css">
-  <link rel="stylesheet" type="text/css" href="//cdn.datatables.net/plug-ins/be7019ee387/integration/jqueryui/dataTables.jqueryui.css">
+  <link rel="stylesheet" type="text/css" href="/static/css/jquery-ui.css">
```

Comment:
Hi Parthchandra,

I have reconsidered this.
 
Simply putting these things to source code is indeed not a graceful solution, even this method is just temporarily working in my case.

I agree with your second advice: to make these stuffs downloadable and configurable. But as the typical scene for now is that users can already have them in cache, so may be just customizing and recompiling the code for special requirements like this is more reasonable for users like me (the scripts are not fetchable from my area, the network hangs while accessing cdn, util click "stop" actively. To have both the cdn and local files is therefore not work for me), I am so going to shelve this issue util someone report the same problem. Thank you.